test/e2e/serial/tests/workload_placement_nodesel.go

```diff
@@ -111,23 +115,23 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			// and the other one will not have enough resources on only one numa zone
 			// but will fulfill the node selector filter.
 			targetNodeName, ok := nrtCandidateNames.PopAny()
-			Expect(ok).To(BeTrue(), "cannot select a targe node among %#v", nrtCandidateNames.List())
+			Expect(ok).To(BeTrue(), "cannot select a target node among %#v", nrtCandidateNames.List())
 			By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
 
 			toAlsoLabelNodeName, ok := nrtCandidateNames.PopAny()
-			Expect(ok).To(BeTrue(), "cannot select a targe node among %#v", nrtCandidateNames.List())
+			Expect(ok).To(BeTrue(), "cannot select a target node among %#v", nrtCandidateNames.List())
 			By(fmt.Sprintf("selecting node to schedule the pod: %q", toAlsoLabelNodeName))
 
 			labelName := "size"
 			labelValue := "medium"
 			By(fmt.Sprintf("Labeling nodes %q and %q with label %q:%q", targetNodeName, toAlsoLabelNodeName, labelName, labelValue))
 
-			unlabelOneFunc, err := labelNodeWithValue(fxt.Client, labelName, labelName, targetNodeName)
+			unlabelOneFunc, err := labelNodeWithValue(fxt.Client, labelName, labelValue, targetNodeName)
```

Comment:
there is similar wrong labeling for node `toAlsoLabelNodeName`

Reply:
good catch, fixed!